samples/bot-calling-meeting/csharp/Source/CallingBotSample/AdaptiveCards/AdaptiveCardFactory.cs

```diff
@@ -10,6 +11,10 @@ namespace CallingBotSample.AdaptiveCards
 {
     public class AdaptiveCardFactory : IAdaptiveCardFactory
     {
+        // Date Time must be in RFC 3389 format https://learn.microsoft.com/en-us/adaptive-cards/authoring-cards/text-features#datetime-function-rules
+        public const string DATE_TIME_STRING_FORMAT = "yyyy-MM-dd'T'HH:mm:ssK";
```

Comment:
I think the .NET naming convention for constants is PascalCase, so `DateTimeStringFormat`

Reply:
Convert comment to doc string comment